### PR TITLE
MCOL-4068 Introduce GET EM endpoint.

### PR DESCRIPTION
**Pull request by drrtuy**

### CODE REVIEW

File: cmapi_server/controllers/dispatcher.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-from cmapi_server.controllers.endpoints import StatusController, ConfigController
-from cmapi_server.controllers.endpoints import BeginController, CommitController
-from cmapi_server.controllers.endpoints import RollbackController, StartController
-from cmapi_server.controllers.endpoints import ShutdownController
+from cmapi_server.controllers.endpoints import StatusController, ConfigController, \
+BeginController, CommitController, RollbackController, StartController, \
+ShutdownController, ExtentMapController
 
-_version = '0.3.0'
+_version = '0.4.0'
```
Comment (pleblanc1976) [inline cmapi_server/controllers/dispatcher.py:7]: This is in the cmapi config file now.  Probably still in PR for though.  Can change later.

File: cmapi_server/controllers/endpoints.py [hunk 2]
```diff
@@ -440,3 +442,14 @@ def put_shutdown(self):
...
+
+
+class ExtentMapController:
+    @cherrypy.tools.validate_api_key()
+    def get_em(self):
+        em_current_name = Path(f'{em_path}/BRM_saves_current')
+        em_current_filename = em_current_name.read_text().rstrip()
+        em_current_file = Path(f'{em_path}/{em_current_filename}_em')
+        extent_map = em_current_file.read_bytes()
+        return extent_map
```
Comment (pleblanc1976) [inline cmapi_server/controllers/endpoints.py:454]: This won't work if storagemanager is handling the file IO

File: cmapi_server/controllers/dispatcher.py (hunk 1 above)
Comment (drrtuy) [reply cmapi_server/controllers/dispatcher.py:7]: Kindly see the comments in that PR.

File: cmapi_server/controllers/endpoints.py (hunk 2 above)
Comment (drrtuy) [reply cmapi_server/controllers/endpoints.py:454]: This endpoint must't be involved with S3 so client side,  namely mcs-loadbrm.py won't interact with the endpoint.

**Comment by drrtuy** [comment]
Looking at this endpoint usage scenarios that are: start of the cluster and nodes list manupulation I can say the endpoint will happen only when cluster doesn't work. Despite this fact I will add dbrm locking prior the operation and subsequent release b/c it looks appropriate.

**Pull request merged**